src/App.js

```diff
@@ -60,13 +64,41 @@ class App extends Component {
       .catch(error => this.setState({error: error}))
     }
 
+  clearInput() {
```

Comment:
This is a p[iece of functionality that I would have never thought of implementing, good catch!